web/pdf_print_service.js

```diff
@@ -26,10 +27,9 @@ function renderPage(activeServiceOnEntry, pdfDocument, pageNumber, size) {
   let scratchCanvas = activeService.scratchCanvas;
 
   // The size of the canvas in pixels for printing.
-  const PRINT_RESOLUTION = 150;
-  const PRINT_UNITS = PRINT_RESOLUTION / 72.0;
-  scratchCanvas.width = Math.floor(size.width * PRINT_UNITS);
-  scratchCanvas.height = Math.floor(size.height * PRINT_UNITS);
+  let printUnits = AppOptions.get('printResolution') / 72.0;
```

Comment:
Please don't change `const` to `let` unless actually necessary, and don't unnecessarily rename variables (it only increases the size of the diff).

Just change this back, and format it like this instead:
```javascript
const PRINT_RESOLUTION = AppOptions.get('printResolution') || 150;
const PRINT_UNITS = PRINT_RESOLUTION / 72.0;
```

Reply:
Fixed that, too. I had misconceptions about the use of `const` in Javascript, so I learned something, too.